sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -838,9 +887,9 @@ def push(self, request):
     if not request.instruction_id:
       self._uid_counter += 1
       request.instruction_id = 'control_%s' % self._uid_counter
-    logging.debug('CONTROL REQUEST %s', request)
+    logging.debug('CONTROL REQUEST ID %s', request.instruction_id)
```

Comment:
When debugging fnapi request and response, the instruction id may not be enough for that purpose. Sometimes instruction type + instruction id would be more helpful. But perhaps we can delete these debug logs and let dev adds them if needed. wdyt?

Reply:
Done.